unstructured/partition/lang.py

```diff
@@ -146,8 +146,17 @@ def prepare_languages_for_tesseract(languages: Optional[List[str]] = ["eng"]):
     if languages is None:
         raise ValueError("`languages` can not be `None`")
     converted_languages = list(
-        filter(None, [convert_language_to_tesseract(lang) for lang in languages]),
+        filter(
+            lambda x: x is not None and x != "",
+            [convert_language_to_tesseract(lang) for lang in languages],
+        ),
     )
+    # Remove duplicates from the list but keep the original order
+    converted_languages = list(dict.fromkeys(converted_languages))
+    if len(converted_languages) == 0:
```

Comment:
If no languages are found, is there any way to proceed with the tesseract call anyway? Could we set a default of English since that's what tesseract uses as default (iirc), instead of raising an error? I would rather the user not be hung up at this step due to language conversion/detection and still be able to process through to tesseract

Reply:
good call :) I think i will change this to a logger warning info and use language detection if no languages are converted, and worse case we use eng default